The docs update, mostly related to how to package the extension and h… (#29)
pyrsia/pyrsia-vscode-extension: README.md

> The Pyrsia extension is not available in the VS Code store yet, it's necessary to manually install the extension as described below.

Reviewer: ```suggestion
The Pyrsia extension is not yet listed in the VS Code marketplace. For now it can still be installed manually following the instructions outlined below.
```

### Install extension (side-load extension)

- Download the latest version of the extension (vsix file) from the [project release page](https://github.com/pyrsia/pyrsia-vscode-extension/releases).

- Close the IDE and install the extension using the `vsix` file downloaded in the previous step.

    ```sh
    code --install-extension <PYRSIA_VSIX_FILE_PATH>

>     ```

Reviewer: As mentioned in https://github.com/pyrsia/pyrsia-vscode-extension/issues/31 good to have uninstall extension command for vscode.

    DONE  Packaged: /home/joed/repositories/pyrsia-vscode-extension/pyrsia-integration-0.0.1.vsix (960 files, 2.2MB)
    ```

- Close the IDE and install the extension using the `vsix` file path from the previous step.

    ```sh
    code --install-extension <PYRSIA_VSIX_FILE_PATH>
    ```
  
### Uninstall (side-load)

> - Find the extension in the list of the extensions (look for "pyrsia", for example `undefined_publisher.pyrsia-integration`).

Reviewer: ```suggestion
- Find the Pyrsia extension from VS Code Extensions list  (look for **Pyrsia** - _Pyrsia VS Code Extension_ by `undefined_publisher` ).
```
